Formula/a/asymptote.rb

```diff
@@ -70,3 +74,17 @@ def install
     assert_predicate testpath/"line.pdf", :exist?
   end
 end
+
+__END__
+diff --git a/Makefile.in b/Makefile.in
+index 3184b27..5b2a11b 100644
+--- a/Makefile.in
++++ b/Makefile.in
+@@ -328,7 +328,6 @@ install-asy: asy sty $(PYFILES)
+ 	-${INSTALL} -p -m 644 GUI/xasyversion/*.py $(GUIdir)/xasyversion
+ 	${INSTALL} -p -m 644 GUI/configs/*.cson $(GUIdir)/configs
+ 	${INSTALL} -p -m 644 GUI/res/icons/*.svg $(GUIdir)/res/icons
+-	ln -sf @datadir@/asymptote/GUI/xasy.py $(bindir)/$(XNAME)
```

Comment:
We should probably make this executable instead of skipping the symlink, since that seems to be what upstream are inclined to do.

Reply:
I have no idea for that, can you give me a hint how to do that? OR any help is really welcome.

Reply:
Something like
```diff
diff --git a/Formula/a/asymptote.rb b/Formula/a/asymptote.rb
index d7ba6e46374..1bd175d028a 100644
--- a/Formula/a/asymptote.rb
+++ b/Formula/a/asymptote.rb
@@ -55,6 +55,8 @@ class Asymptote < Formula
     system "make", "asy-keywords.el"
     system "make", "install-asy"
 
+    # https://github.com/vectorgraphics/asymptote/issues/497
+    chmod "+x", bin/"xasy"
     doc.install resource("manual")
     (share/"emacs/site-lisp").install_symlink pkgshare
   end
```
Should work if you drop the patch.

Reply:
Oh, that simple. I will test and update soon. Thank you @carlocab !

Reply:
I was thought about the problem is installing non-executable to the bin and so tried to remove, but giving permission is enough.